generate.py

```diff
@@ -0,0 +1,39 @@
+from transformers import AutoModelForCausalLM, AutoTokenizer
+path = '/path/to/model/checkpoint/directory'
+model_name = path
+tokenizer = AutoTokenizer.from_pretrained(model_name)
+model = AutoModelForCausalLM.from_pretrained(model_name)
+
+import torch
+device = torch.device('cuda:0')
+model.to(device)
+
+def generate_text(model, tokenizer, input_text, device):
+    with torch.no_grad():
+        input_tokens = tokenizer.encode(input_text, return_tensors="pt").to(device)
+        output_tokens = model.generate(
+            input_tokens,
+            max_length=300,  # Controls the maximum length of the generated text
```

Comment:
Can we make these parameters that you can pass on the command line, so you can run with `python`?